Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+# How to label a Pull Request
+
+
+We use labels in our Pull Requests to be able to categorize and prioritize our work. Some specific labels are also used to trigger our CI jobs.
+
+
+## 1. The labels
+
+At the moment we have 26 labels in our repository. They can be separated into 4 groups:
+
+- [Product verticals](#product-verticals)
+
+- [Project horizontals](#project-horizontals)
+
+- [Branding](#branding)
+
+- [Status](#status)
+
+- [Emergency](#emergency)
+
+- [Extra](#extra)
+
+
+### <a name="product-verticals"></a>Product verticals
+A product vertical corresponds to a big feature of the product. 
+
+- <span style="background-color:#2BBBBB; color:black; font-weight:600">Health Management 🍄</span> - any work done in the health management vertical
+- <span style="background-color:#5319e7; color:white; font-weight:600">Healthcheck 👩‍⚕️</span> - any work done in the healthcheck vertical
+- <span style="background-color:#25fcb8; color:black; font-weight:600">Payments 💰</span> - any work done in the payments vertical
+- <span style="background-color:#d3cb34; color:black; font-weight:600">Prescriptions 💊</span> - any work done in the prescriptions vertical
+- <span style="background-color:#f9d0c4; color:black; font-weight:600">SDK 💸</span> - any work done in the SDK vertical
+- <span style="background-color:#006b75; color:white; font-weight:600">Triage UI 🏥</span> - any work done in the Triage UI vertical
+
+### <a name="project-horizontals"></a>Project horizontals
+A project horizontal corresponds to an area of the project that has an horizontal impact, means it has impact in most or all the verticals in the project.
+
+- <span style="background-color:#d4c5f9; color:black; font-weight:600">Automation Tests 🤖</span> - introduction of automation tests
+- <span style="background-color:#a3f7e2; color:black; font-weight:600">Linter 🧹</span> - style guide work
+- <span style="background-color:#fcfc50; color:black; font-weight:600">Localisation 🌐</span> - updating localizable strings or working in the localization system of the project
+- <span style="background-color:#8631bf; color:white; font-weight:600">Infrastructure 🛠</span> - working in the foundation of the project.
```

Comment:
This one needs clarification. Does that involve refactoring some code without changing functionality (or does that one falls under "Housekeeping")? Does that involve architecture restructuration (but how to differentiate that from refactoring)? Does that involve only tooling (CI, fastlane, Danger, linter, ...)?